.happy/terraform/modules/sfn_config/nextstrain.wdl

```diff
@@ -68,36 +68,37 @@ task nextstrain_workflow {
     )
 
     # set up ncov
-    mkdir -p /ncov/my_profiles/aspen /ncov/results
-    cd /ncov
-    git init
-    git fetch --depth 1 git://github.com/nextstrain/ncov.git
-    git checkout FETCH_HEAD
-    ncov_git_rev=$(git rev-parse HEAD)
+    mkdir -p ncov/my_profiles/aspen ncov/results
+    (cd ncov &&
+     git init &&
+     git fetch --depth 1 git://github.com/nextstrain/ncov.git &&
+     git checkout FETCH_HEAD
+    )
+    ncov_git_rev=$(cd ncov && git rev-parse HEAD)
 
-    cp /usr/src/app/aspen/workflows/nextstrain_run/nextstrain_profile/* /ncov/my_profiles/aspen/
+    cp /usr/src/app/aspen/workflows/nextstrain_run/nextstrain_profile/* ncov/my_profiles/aspen/
 
     # dump the sequences, metadata, and builds.yaml for a run out to disk.
     aligned_gisaid_location=$(
         python3 /usr/src/app/aspen/workflows/nextstrain_run/export.py \
                --phylo-run-id "${workflow_id}"                        \
-               --sequences /ncov/data/sequences_aspen.fasta           \
-               --metadata /ncov/data/metadata_aspen.tsv               \
-               --builds-file /ncov/my_profiles/aspen/builds.yaml      \
+               --sequences ncov/data/sequences_aspen.fasta            \
+               --metadata ncov/data/metadata_aspen.tsv                \
+               --builds-file ncov/my_profiles/aspen/builds.yaml       \
     )
     aligned_gisaid_s3_bucket=$(echo "${aligned_gisaid_location}" | jq -r .bucket)
     aligned_gisaid_sequences_s3_key=$(echo "${aligned_gisaid_location}" | jq -r .sequences_key)
     aligned_gisaid_metadata_s3_key=$(echo "${aligned_gisaid_location}" | jq -r .metadata_key)
 
     # fetch the gisaid dataset
-    aws s3 cp --no-progress "s3://${aligned_gisaid_s3_bucket}/${aligned_gisaid_sequences_s3_key}" - | zstdmt -d | xz -2 > /ncov/results/aligned_gisaid.fasta.xz
-    aws s3 cp --no-progress "s3://${aligned_gisaid_s3_bucket}/${aligned_gisaid_metadata_s3_key}" /ncov/data/metadata_gisaid.tsv
+    aws s3 cp --no-progress "s3://${aligned_gisaid_s3_bucket}/${aligned_gisaid_sequences_s3_key}" - | zstdmt -d | xz -2 > ncov/results/aligned_gisaid.fasta.xz
+    aws s3 cp --no-progress "s3://${aligned_gisaid_s3_bucket}/${aligned_gisaid_metadata_s3_key}" ncov/data/metadata_gisaid.tsv
 
-    snakemake --printshellcmds auspice/ncov_aspen.json --profile my_profiles/aspen/  --resources=mem_mb=312320 || aws s3 cp /ncov/.snakemake/log/ "s3://${aspen_s3_db_bucket}/phylo_run/${build_id}/" --recursive
+    (cd ncov && snakemake --printshellcmds auspice/ncov_aspen.json --profile my_profiles/aspen/ --resources=mem_mb=312320) || aws s3 cp ncov/.snakemake/log/ "s3://${aspen_s3_db_bucket}/phylo_run/${build_id}/" --recursive
```

Comment:
would `cd ncov` here influence the following `aws s3 cp ncov/...` and in line 101, 119

Reply:
No, it's in a subshell, so it doesn't affect the rest.